### PR TITLE
Update wizard overall design and reinstate sub-header text

### PR DESCRIPTION
**Pull request by thomasguillot**
Also reorganise some wizards: 

### All Submissions:

* [x] Have you followed the [Newspack Contributing guideline](https://github.com/Automattic/newspack-plugin/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) and [VIP Go coding standards](https://vip.wordpress.com/documentation/vip-go/code-review-blockers-warnings-notices/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

This is a fairly big PR.

* The display of a sub-header text has been reinstated.
* By doing so I took this opportunity to rework/reorganise some of our wizards, including seo, health check, engagement, advertising.

### How to test the changes in this Pull Request:

1. Switch to this branch
2. Test ALL the screens

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?

### CODE REVIEW

File: assets/wizards/seo/views/separator/index.js [hunk 1]
```diff
@@ -1,91 +0,0 @@
...
-	render() {
-		const { data, onChange } = this.props;
-		const { titleSeparator } = data;
-		return (
-			<Fragment>
-				<SectionHeader
-					title={ __( 'Title separator', 'newspack' ) }
-					description={ () => (
-						<>
-							{ __( 'Choose the symbol to use as your title separator', 'newspack' ) }
```
Comment (adekbadek) [inline assets/wizards/seo/views/separator/index.js:56]: This part was not moved to the new view – is this intentional?

**Review by adekbadek — CHANGES_REQUESTED** [review]
When on a "Required plugin" version of the (Ad) wizard, padding is missing:

image

File: assets/wizards/seo/views/separator/index.js (hunk 1 above)
Comment (thomasguillot) [reply assets/wizards/seo/views/separator/index.js:56]: Yes, I didn't see a point in keeping this one. Most publishers use the default setting and never change this.

File: assets/wizards/advertising/views/ad-units/index.js [hunk 2]
```diff
@@ -186,13 +186,13 @@ const AdUnits = ( {
...
 											icon={ pencil }
-											label={ __( 'Edit the ad unit', 'newspack' ) }
+											label={ __( 'Edit Ad Unit', 'newspack' ) }
 											{ ...buttonProps }
 										/>
 										<Button
 											onClick={ () => onDelete( adUnit.id ) }
 											icon={ trash }
-											label={ __( 'Archive the ad unit', 'newspack' ) }
+											label={ __( 'Delete Ad Unit', 'newspack' ) }
```
[the remark this answers is not in the document]
Comment (thomasguillot) [reply assets/wizards/advertising/views/ad-units/index.js:195]: Oh ok so we need to update the icon then!

**Comment by thomasguillot** [comment]
> When on a "Required plugin" version of the (Ad) wizard, padding is missing:

fixed in https://github.com/Automattic/newspack-plugin/pull/1457/commits/a6371cd61165bec8686e26d9f6f094ac054ef420

File: assets/wizards/advertising/views/ad-units/index.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (thomasguillot) [reply assets/wizards/advertising/views/ad-units/index.js]: Fixed in https://github.com/Automattic/newspack-plugin/pull/1457/commits/a1e7d376560a92d0d1a4a96f3008a1ab331b8214

**Review by adekbadek — APPROVED** [review]

**Pull request merged**